website/docs/jellyfish/api/account.md

```diff
@@ -216,7 +216,8 @@ enum DfTxType {
   CREATE_POOL_PAIR = 'p',
   UPDATE_POOL_PAIR = 'u',
   SET_GOV_VARIABLE = 'G',
-  AUTO_AUTH_PREP = 'A'
+  AUTO_AUTH_PREP = 'A',
+  NONE = '0'
```

Comment:
I am not sure if this is explicitly for burn history only. Is it a legit filter option for other account history listing since it is actually not a custom DfTx in the first place.

Reply:
I don't fully understand the implication of your question. 
It's a valid value for filtering by txtype though

Reply:
It is a valid DfTxType on the masternode side as can be seen in `ain` `src/masternodes/mn_checks.h`